src/main/scala/mesosphere/marathon/core/launcher/impl/InstanceOpFactoryHelper.scala

```diff
@@ -19,36 +17,25 @@ class InstanceOpFactoryHelper(
 
   def provision(
```

Comment:
I'm not sure this helper yields much value anymore.

Reply:
I would drop this